src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE books (
+    id INT NOT NULL  AUTO_INCREMENT,
```

Comment:
Read checklist carefully: When creating a table in MySQL, use bigint column type for storing id.